run-tests.sh

```diff
@@ -1,14 +1,18 @@
 #!/usr/bin/bash
 set -euo pipefail
 
+SHELLCHECK_SEVERITY=${SHELLCHECK_SEVERITY:-warning}
 
 run_test() {
-  podman run -it -v "$(pwd)":/github/workspace:z --env "GITHUB_WORKSPACE=/github/workspace" koji.test "$1"
+  podman run --rm -it -v "$(pwd)":/github/workspace:z --env "GITHUB_WORKSPACE=/github/workspace" koji.test "$1"
```

Comment:
Did you want to ad this `--rm`? Seems unrelated to this commit.

Reply:
:shushing_face:  :upside_down_face: Should be fixed ;)